pipenv/core.py

```diff
@@ -1028,23 +1028,6 @@ def do_lock(
         deps = convert_deps_to_pip(
             settings["packages"], project, r=False, include_index=True
         )
-        results = venv_resolve_deps(
-            deps,
-            which=which,
-            project=project,
-            clear=clear,
-            pre=pre,
-            allow_global=system,
-            pypi_mirror=pypi_mirror,
-        )
-        # Add dependencies to lockfile.
-        for dep in results:
-            is_top_level = dep["name"] in settings["packages"]
-            pipfile_entry = settings["packages"][dep["name"]] if is_top_level else None
-            dep_lockfile = clean_resolved_dep(
-                dep, is_top_level=is_top_level, pipfile_entry=pipfile_entry
-            )
-            lockfile[settings["lockfile_key"]].update(dep_lockfile)
         # Add refs for VCS installs.
         # TODO: be smarter about this.
         vcs_reqs, vcs_lockfile = get_vcs_deps(
```

Comment:
The get_vcs_deps is also a slow function, but without spinner. The pipenv will looks like freeze in this step. Can it be moved to venv_resolve_deps or wrap them into one function with spinner?

It is not a big issue. Just it will make the UI better.

Reply:
sure thing